api/pom.xml

```diff
@@ -9,7 +9,17 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>ApiLayer</artifactId>
+    <artifactId>api</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>cz.idomatojde</groupId>
+            <artifactId>dao</artifactId>
+            <version>1.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```

Comment:
API should not depend directly on DAO

Reply:
We need to have this dependency for Category enum. I can create another module for this class but I am not sure if that is the correct solution